src/main/java/org/alliancegenome/curation_api/controllers/crud/VocabularyCrudController.java

```diff
@@ -26,6 +26,16 @@ protected void init() {
     public ObjectResponse<Vocabulary> get(Long id) {
         return vocabularyService.get(id);
     }
+    
+    @Override
+    public ObjectResponse<Vocabulary> get(String name) {
+        SearchResponse<Vocabulary> ret = findByField("name", name);
+        if(ret != null && ret.getTotalResults() == 1) {
+            return new ObjectResponse<>(ret.getResults().get(0));
+        } else {
+            return new ObjectResponse<>();
+        }
+    }
 
     @Override
     public ObjectListResponse<VocabularyTerm> getTerm(Long id) {
```

Comment:
This method `getTerm` should be renamed to `getTerms` I know thats not part of this PR but anyway.

Reply:
Done